### PR TITLE
config: Debug log ignored sections (fixes: #694)

### PR DESCRIPTION
**Pull request by adamcik**

### CODE REVIEW

File: mopidy/config/__init__.py [hunk 1]
```diff
@@ -182,6 +183,12 @@ def _validate(raw_config, schemas):
             errors[schema.name] = error
         if result:
             config[schema.name] = result
+        if schema.name in sections:
+            sections.remove(schema.name)
```
Comment (jodal) [inline mopidy/config/__init__.py]: I believe `sections.discard(schema.name)` does the same without the if.

File: mopidy/config/__init__.py [hunk 2]
```diff
@@ -182,6 +183,12 @@ def _validate(raw_config, schemas):
...
+
+    for section in sections:
+        logger.debug('Ignoring unknown config section: %s', section)
```
Comment (jodal) [inline mopidy/config/__init__.py]: Are you sure this shouldn't be at least at info level or maybe even on warning level?

Comment (adamcik) [inline mopidy/config/__init__.py]: Do we need the noise / red herrings? My other thought was to propagate this up the chain in errors or as an extra return value, but this is just the quick shortcut to at least make it a bit discoverable what is going on. I've also dropped the idea of having a config to disable this.

Comment (jodal) [inline mopidy/config/__init__.py]: Leaving it at debug level also makes it more usable for e.g. Pi Musicbox to use Mopidy's config as a general config store.

I'm OK with this as is.

**Comment by adamcik** [comment]
Fixed.

**Comment by jodal** [comment]
Missing changelog entry

**Comment by adamcik** [comment]
And now with a changelog entry :-)

**Pull request merged**